### PR TITLE
add guards for invalid location or dimension in cuboid.tsx

### PR DESCRIPTION
**Pull request by sashankaryal**
## What changes are proposed in this pull request?

add guards for invalid location or dimension in cuboid.tsx

## How is this patch tested? If it is not, please explain why.

Locally

### Is this a user-facing change that should be mentioned in the release notes?

-   [x] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [x] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

**Review by benjaminpkane — APPROVED** [review]
LGTM!

File: app/packages/looker-3d/src/overlays/cuboid.tsx [hunk 1]
```diff
@@ -21,6 +21,13 @@ export const Cuboid = ({
   color,
   useLegacyCoordinates,
 }: CuboidProps) => {
+  const geo = React.useMemo(
+    () => dimensions && new THREE.BoxGeometry(...dimensions),
+    [dimensions]
+  );
+
+  if (!location || !dimensions) return null;
```
Comment (benjaminpkane) [inline app/packages/looker-3d/src/overlays/cuboid.tsx:29]: Returning here changes the number of hooks, no? Maybe needs a tweak

Comment (sashankaryal) [reply app/packages/looker-3d/src/overlays/cuboid.tsx:29]: There's only one hook here that I see and that comes before the return statement.

Comment (benjaminpkane) [reply app/packages/looker-3d/src/overlays/cuboid.tsx:29]: Oh only one hook, my mistake

**Pull request merged**